### PR TITLE
Add conversion of booleans and a test

### PR DESCRIPTION
**Pull request by royreiss**
Hi, 
postgres-array isn't converting boolean arrays correctly.  Arrays of type BOOL[] are brought into node as strings.
Thanks!

### CODE REVIEW

**Review by bendrucker — CHANGES_REQUESTED** [review]
Can you clean up the final change by using typeof?

File: index.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ ArrayParser.prototype.newEntry = function (includeEmpty) {
     if(entry === 'false'){
       entry = false;
     }
-    if (entry !== null && entry !== true && entry !== false) entry = this.transform(entry)
+    if (entry !== null && typeof(entry) !== 'boolean') entry = this.transform(entry)
```
Comment (bendrucker) [inline index.js]: typeof is an opertor, not a function (`typeof entry`)

**Review by bendrucker — CHANGES_REQUESTED** [review]
see typeof comment

**Comment by royreiss** [comment]
Sorry about that!  All this time I've been using it with parens <facepalm>

**Review by bendrucker — CHANGES_REQUESTED** [review]
Check Travis, there are a few errors being thrown by standard. After that, LGTM.

**Comment by bendrucker** [comment]
Hey, wasn't thinking when I reviewed this. This code doesn't belong here—I'm reverting. That's what the `transform` is for. Belongs in node-pg-types.

**Comment by royreiss** [comment]
That makes a lot more sense :)  Thanks for submitting that!

**Pull request merged**